src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private final Object[] keys = new Object[MAX_LENGTH];
+    private final Object[] values = new Object[MAX_LENGTH];
+    private int size;
+
+    public boolean isEquel(Object firstObject, Object secondObject) {
```

Comment:
```suggestion
    private boolean isEqual(Object firstObject, Object secondObject) {
```

And move it to the end of the class, first the public methods, then the private ones